src/main/java/frc/robot/subsystems/gpm/Intake.java

```diff
@@ -27,20 +28,24 @@ public double getPower() {
         }
     }
 
-    private final CANSparkFlex motor;
+    public final CANSparkMax motor;
     private final DigitalInput sensor;
 
     private int countSim = 0;
     private DIOSim IntakeSensorDioSim;
     private  boolean foo = true;
 
+    // private XboxController m_gc;
+
     private Mode mode;
 
 
-    public Intake() {
-        motor = new CANSparkFlex(IntakeConstants.MOTOR_ID, CANSparkLowLevel.MotorType.kBrushless);
+    public Intake() { //XboxController gc
+        motor = new CANSparkMax(IntakeConstants.MOTOR_ID, CANSparkLowLevel.MotorType.kBrushless);
         sensor = new DigitalInput(IntakeConstants.SENSOR_ID);
-        mode = Mode.DISABLED;
+        mode = Mode.INTAKE;
```

Comment:
When you deploy, it should be default disabled. Also, remove all commented out references to a controller.

Reply:
done